Napzakmarket-iOS/Napzakmarket-iOS/Presentation/Mypage/View/MarketView.swift

```diff
@@ -0,0 +1,219 @@
+            } label: {
+                Image(.chipGenre)
+                    .resizable()
+                    .frame(width: 67, height: 33)
+            }
+            .padding(.leading, 20)
+
+            Button {
+                print("품절 제외 필터 선택")
+            } label: {
+                Image(.chipSoldout)
+                    .resizable()
+                    .frame(width: 69, height: 33)
+            }
+            
+            if selectedIndex == 0 {
+                Button {
+                    print("미개봉 필터 선택")
+                } label: {
+                    Image(.chipUnopen)
+                        .resizable()
+                        .frame(width: 59, height: 33)
+                }
+            }
+            Spacer()
+        }
+        .frame(height: 53)
+        .background(Color.napzakGrayScale(.gray50))
+    }
+   
+    private var navigationSection: some View {
+        ZStack(alignment: .top) {
+            Color.napzakGrayScale(.gray200)
+                .edgesIgnoringSafeArea(.top)
+                .frame(height: 138)
+           
+            HStack(alignment: .center, spacing: 0) {
+                Spacer()
+                Image("img_market_bg_character")
+                    .resizable()
+                    .frame(width: 138, height: 111, alignment: .trailing)
+                    .padding(.top, 27)
+            }
+           
+            HStack {
+                Button(action: {
+                    dismiss()
+                }) {
+                    Image("ic_back")
+                        .foregroundColor(Color.napzakGrayScale(.gray900))
+                        .padding(.leading, 20)
+                }
+                Spacer()
+            }
+            .padding(.horizontal, 16)
+            .padding(.vertical, 12)
+        }
+        .frame(maxWidth: .infinity)
+    }
+   
+    private var profileSection: some View {
+        VStack(spacing: 12) {
+            Image("img_profile_md")
+                .resizable()
+                .scaledToFit()
+                .frame(width: 88, height: 88)
+                .overlay(Circle().stroke(Color.white, lineWidth: 4))
+                .padding(.top, -63)
+           
+            Text("아요핑들")
+                .font(.napzakFont(.title2Bold20))
+                .applyNapzakTextStyle(napzakFontStyle: .title2Bold20)
+                .foregroundStyle(Color.napzakGrayScale(.gray900))
+                .padding(.top, -8)
+           
+            tagListView
+                .padding(.top, -5)
+           
+            Text("마이멜로디, 시나모롤 제일 좋아합니다 :) 해당 장르 상품들 판매 및 제시 채팅 언제든 환영합니다!")
+                .multilineTextAlignment(.center)
+                .padding(.horizontal, 20)
+                .padding(.vertical, 12)
+                .padding(.top, -12)
+                .font(.napzakFont(.caption3Medium12))
+                .applyNapzakTextStyle(napzakFontStyle: .caption3Medium12)
+                .foregroundStyle(Color.napzakGrayScale(.gray700))
+        }
+    }
```

Comment:
P1

```suggestion
        VStack(spacing: 0) {
            Image("img_profile_md")
                .resizable()
                .scaledToFit()
                .frame(width: 88, height: 88)
                .overlay(Circle().stroke(Color.white, lineWidth: 4))
                .padding(.top, -55)
           
            Text("아요핑들")
                .font(.napzakFont(.title2Bold20))
                .applyNapzakTextStyle(napzakFontStyle: .title2Bold20)
                .foregroundStyle(Color.napzakGrayScale(.gray900))
                .padding(.top, 6)
           
            tagListView
                .padding(.top, 8)
           
            Text("마이멜로디, 시나모롤 제일 좋아합니다 :) 해당 장르 상품들 판매 및 제시 채팅 언제든 환영합니다!")
                .multilineTextAlignment(.center)
                .padding(.horizontal, 20)
                .padding(.top, 12)
                .font(.napzakFont(.caption3Medium12))
                .applyNapzakTextStyle(napzakFontStyle: .caption3Medium12)
                .foregroundStyle(Color.napzakGrayScale(.gray700))        }
    }
```
여기도 spacing 값 때문에 패딩 값이 복잡하게 들어가있는데 spacing을 아예 0으로 지정하고 top padding 만 각각 사용해서 간격을 요렇게 바꿔주면 좋을 듯 합니다~!